### DISCUSSION TITLE
Windows 11 clock is clipped with small taskbar icon

### DISCUSSION THREAD

**Discussion by kohsh** [valinet/ExplorerPatcher#832] [General]
![image](https://user-images.githubusercontent.com/5834110/153287018-36828e11-4db6-469d-acb2-1a2cae704fe4.png)

**Comment by kohsh** [comment]
Repos with latest stable release: 22000.469.41.16
Unsure if this is a regression or not. 
Thanks.

**Reply by valinet** [reply]
It’s not. This is Windows’ behavior, EP doesn’t have anything to do with it.
